helpers/post_proc_gard.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import xarray as xr
+import numpy as np
+from scipy import stats
+
+varname = "PRECT"
+output_base = "output/gard_out_"
+
+has_pop = True
+
+mean_file = output_base+varname+".nc"
+err_file = output_base+varname+"_errors.nc"
+pop_file = output_base+varname+"_logistic.nc"
+
+# scrfs_file = "conus_scrfs_150yrs.nc"
+scrfs_file = "AR_scrfs.nc"
+output_file = "processed_{}.nc".format(varname)
+
+
+def main():
+
+    # open the primary data files
+    mean = xr.open_dataset(mean_file)
+    err = xr.open_dataset(err_file)
+    scrfs = xr.open_dataset(scrfs_file)
+
+    # apply the transform that was used within GARD
+    d1 = np.cbrt(mean[varname])
+
+    # get the random variables into a format that is appropriate in case the shapes don't match
+    rand_tmp = scrfs.p_rand_uniform[:d1.shape[0], :d1.shape[1], :d1.shape[2]]
+
+    # if there is a probability of precipitation (PoP) or other threshold value specified
+    # then we need to do some extra work to use that prediction
+    if has_pop:
+        # load the PoP dataset
+        pop = xr.open_dataset(pop_file)
+        popval = pop[varname+"_exceedence_probability"]
+
+        # figure out where the probability was not exceeded
+        noprecip = rand_tmp.values < (1-popval.values)
+        # rescale the random numbers to account for the limited portion of the cdf they should occupy
+        rand_tmp.values -= (1-popval.values)
+        rand_tmp /= popval.values
```

Comment:
I'm wondering if we can just use what is in https://github.com/jhamman/storylines/blob/refactor/storylines/tools/post_process_gard_output.py#L79

That code does what this does, plus has some performance optimizations. I'd be totally fine moving that module over here.